1-syntax-basic-constructs/basic.js

```diff
@@ -0,0 +1,6 @@
+let userAge = parseInt(prompt("Write your age in the space below"));
+alert(areYouAnAdult(userAge));
```

Comment:
```suggestion
let adult = areYouAnAdult(userAge);
alert(adult);
```

Reply:
I know that works perfect in your way. But if we want to reuse the return value in the future, it's better to save it in a variable.